src/main/groovy/org/shipkit/internal/notes/vcs/GitOriginRepoProvider.java

```diff
@@ -14,9 +14,17 @@ public GitOriginRepoProvider(ProcessRunner processRunner){
      * fetches remote url for git origin and returns it in format "user/repo", eg. "mockito/shipkit"
      */
     public String getOriginGitRepo(){
-        return processRunner.run("git", "remote", "get-url", "origin") // for GitHub returns git@github.com:user/repo.git
-                    .trim()
+        // for GitHub returns git@github.com:user/repo.git or https://github.com/user/repo.git
+        String remote = processRunner.run("git", "remote", "get-url", "origin").trim();
+
+        if(remote.startsWith("git")){
+            return remote
                     .replaceFirst("^.*?:", "") // remove everything before ':'
                     .replaceFirst(".git$", ""); // remove .git suffix
+        } else{
```

Comment:
I was thinking if we should throw an exception when output for process is unfamiliar. However, I think it's easier and safer if we don't throw anything. Just like in my case, I could just tweak the repo by hand.